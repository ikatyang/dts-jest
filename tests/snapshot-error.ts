declare const snapshot_error: (v: {a: {b: {c: string}}}) => any;

// @dts-jest
snapshot_error(1);

// @dts-jest
snapshot_error({a: {b: 1}});

// @dts-jest
snapshot_error({a: {b: {c: 1}}});
