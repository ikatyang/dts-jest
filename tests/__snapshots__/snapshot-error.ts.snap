// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot_error({a: {b: {c: 1}}}) 1`] = `
"Argument of type '{ a: { b: { c: number; }; }; }' is not assignable to parameter of type '{ a: { b: { c: string; }; }; }'.
  Types of property 'a' are incompatible.
    Type '{ b: { c: number; }; }' is not assignable to type '{ b: { c: string; }; }'.
      Types of property 'b' are incompatible.
        Type '{ c: number; }' is not assignable to type '{ c: string; }'.
          Types of property 'c' are incompatible.
            Type 'number' is not assignable to type 'string'."
`;

exports[`snapshot_error({a: {b: 1}}) 1`] = `
"Argument of type '{ a: { b: number; }; }' is not assignable to parameter of type '{ a: { b: { c: string; }; }; }'.
  Types of property 'a' are incompatible.
    Type '{ b: number; }' is not assignable to type '{ b: { c: string; }; }'.
      Types of property 'b' are incompatible.
        Type 'number' is not assignable to type '{ c: string; }'."
`;

exports[`snapshot_error(1) 1`] = `"Argument of type '1' is not assignable to parameter of type '{ a: { b: { c: string; }; }; }'."`;
